packages/atlas/src/components/_navigation/SidenavBase/SidenavBase.styles.ts

```diff
@@ -79,6 +81,8 @@ export const SidebarNavItem = styled.li<ExpandableElementProps>`
   &[data-badge]::after {
     left: ${sizes(12)};
     top: ${sizes(3)};
+    color: blue;
+    background-color: green;
```

Comment:
```suggestion
    background-color: ${cVar('colorBackgroundPrimary')};
```
![Zrzut ekranu 2022-02-3 o 18 31 26](https://user-images.githubusercontent.com/13706246/152396733-829237ee-981d-4d7c-bc61-57643860709f.png)